src/tribol/geom/GeomUtilities.cpp

```diff
@@ -1275,10 +1259,9 @@ void CheckPolySegs( const real* const RESTRICT x, const real* const RESTRICT y,
    {
       if (newIDs[i] == i)
       {
-         // debug error
          if (k > numNewPoints)
          {
-            TRIBOL_ERROR("checkPolySegs: index into polyX/polyY exceeds allocated space");
+            SLIC_ERROR("checkPolySegs: index into polyX/polyY exceeds allocated space");
```

Comment:
Is there a way to handle this (more) gracefully?

Reply:
I fixed this, and another location where this iterator 'may' exceed allocated space (i.e. number of polygonal overlap vertices). I added an enum to catch and report this at a higher level.